packages/truffle-core/lib/commands/index.js

```diff
@@ -9,6 +9,7 @@ module.exports = {
   console: require("./console"),
   develop: require("./develop"),
   create: require("./create"),
+  help: require("./help"),
   install: require("./install"),
   publish: require("./publish"),
   //digest: require("./digest"),
```

Comment:
Also, can we remove this line and the corresponding file from the project?  Tim said it was related to EthPM but did not know if it was relevant with EthPM v2.

Reply:
Yep, totally remove this!

Reply:
Feels good